dir/file2.txt/file.txt

```diff
@@ -0,0 +1,3 @@
+This is some

```

Comment:
Remove this file from the pull request, there is no need for it.

Reply:
![image](https://github.com/mate-academy/py-move-file/assets/111088370/d59264e5-2abf-4f0d-9b0d-a38f78c75c7b)